js/app.js

```diff
@@ -1,8 +1,8 @@
 'use strict';
 
-var hours = ['','6am','7am','8am','9am','10am','11am','12pm','1pm','2pm','3pm','4pm','5pm','6pm','7pm', '8pm','Total'];
+var hours = ['Locations:','6am','7am','8am','9am','10am','11am','12pm','1pm','2pm','3pm','4pm','5pm','6pm','7pm', '8pm','Total'];
 
-var body = document.getElementsByTagName('body')[0];
+var section = document.getElementsByTagName('section')[0];
```

Comment:
This is really ambiguous. Perhaps you should put an ID on the section you want to target and then `getElementByid`?